plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -1090,6 +1098,15 @@ else if (avroSchemaLiteral != null) {
         tableProperties.put("numFiles", "-1");
         tableProperties.put("totalSize", "-1");
 
+        // Extra properties
+        Map<String, String> extraProperties = getExtraProperties(tableMetadata.getProperties());
+        if (extraProperties != null) {
+            tableProperties.put(EXTRA_PROPERTIES, mapJsonCodec(String.class, String.class).toJson(extraProperties));
+            for (Map.Entry<String, String> extraProperty : extraProperties.entrySet()) {
```

Comment:
Can we have an additional check to ensure that - we don't override the existing property here ?

Reply:
Would this be a test that alters the additional properties through hive?  I can't write a test doing this with the hive connector in Trino since it doesn't support `ALTER TABLE <xyz> SET PROPERTIES`.

Reply:
I added a test for when duplicate properties are specified in `extra_properties` when creating a table.

Are you thinking that a user could add duplicate properties via an `ALTER TABLE` statement in hive? Is that possible?

Reply:
I was mentioning if the `extra_properties` could override any of the pre-defined property like file format or table_comment

Reply:
Ah gotcha. No it is not possible for this to occur. I added an extra test for this scenario where we try to override `transactional` table property in `extra_properties`.